README.md

```diff
@@ -5,6 +5,155 @@
 
 ## Introduction
 
-With the popularity of large language models (LLMs), an increasing number of enterprises are seeking private deployments of these models. Choosing the right inference framework is crucial for optimizing performance and resource utilization. This repository is designed to conduct performance benchmark on various LLM inference frameworks, providing users with performance metrics to aid in the selection of the most suitable framework for their needs.
+With the popularity of large language models (LLMs), an increasing number of enterprises are seeking private deployments
+of these models. Choosing the right inference framework is crucial for optimizing performance and resource utilization.
+This repository is designed to conduct performance benchmark on various LLM inference frameworks, providing users with
+performance metrics to aid in the selection of the most suitable framework for their needs.
 
 ## Supported Frameworks
+
+The current code supports vllm and tgi frameworks
```

Comment:
"The current benchmark is taken on vllm 0.2.5 and text-generation-inference 1.3.3"